### PR TITLE
Fixes to the networking spec

### PR DESCRIPTION
**Pull request by AndrewWestberg**
Updating the timestamp section of the mux protocol based on telegram comments from @karknu . 
Updating the 3.2 table to include Transaction Submission protocol for NtN communication. The paragraph above it describes it and I've seen those messages in a NtN wireshark capture.
Also fixing some typographical issues.

### CODE REVIEW

File: docs/network-spec/mux.tex [hunk 1]
```diff
@@ -180,7 +180,7 @@ \section{Life Cycle of a Connection}
   1  & DeltaQ                                & Yes  & Yes \\ \hline
   2  & Chain-Sync instantiated to headers    & Yes  & No \\ \hline
   3  & Block-Fetch                           & Yes  & No  \\ \hline
-  4  & Transaction-Submission                & No   & Yes  \\ \hline
+  4  & Transaction-Submission                & Yes  & Yes  \\ \hline
```
Comment (coot) [inline docs/network-spec/mux.tex:183]: We have two separate `mini-protocols` for transaction submission:
* `LocalTxSubmission` - which is only used for `node-to-client`
* `TxSubmission` - only used by the `node-to-node` protocol.

They are quite a bit different, thus I suggest we have two entries in this table.

**Review by coot — CHANGES_REQUESTED** [review]

File: docs/network-spec/mux.tex (hunk 1 above)
Comment (AndrewWestberg) [reply docs/network-spec/mux.tex:183]: Do these share the same protocol id?

Comment (coot) [reply docs/network-spec/mux.tex:183]: `TxSubmission` has id `4`
`LocaTxSubmission` has id `6`

Comment (AndrewWestberg) [reply docs/network-spec/mux.tex]: Fixed

**Review by coot — APPROVED** [review]
LGTM

**Comment by coot** [comment]
bors merge

**Pull request merged**